e2e/suites/metadataprops.test.ts

```diff
@@ -0,0 +1,166 @@
+/* eslint-disable max-statements */
+
+import { adminLogin, logout } from '../helpers/login';
+import { host } from '../config';
+import proxyMock from '../helpers/proxyMock';
+import insertFixtures from '../helpers/insertFixtures';
+import disableTransitions from '../helpers/disableTransitions';
+import { clearInput, selectDate, scrollTo, waitForNavigation } from '../helpers/formActions';
+
+describe('Metadata Properties', () => {
+  beforeAll(async () => {
+    await insertFixtures();
+    await proxyMock();
+    await disableTransitions();
+  });
+
+  it('should log in as admin then click the settings nav button.', async () => {
+    await adminLogin();
+    await waitForNavigation(expect(page).toClick('a', { text: 'Account settings' }));
```

Comment:
Just curious, why is this needed here?

Reply:
In the prod build (and consequently, on the CI) it picks up the link to early, and the next check fails. See this run:
https://github.com/huridocs/uwazi/runs/2960379218#step:15:44